### PR TITLE
Feature/Add Operations Stakeholder Lab

### PR DESCRIPTION
**Pull request by jkeam**
# Feature - Add Operations Stakeholder Lab

## What
This PR adds lab for operations focused instruction.

## Why
This is for the operations focused lab.

## How
Adds `workshop/content/lab1.6_operator.md`.

## Who
This content is for workshop participants to use.

### CODE REVIEW

File: workshop/content/lab1.6_operator.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+            - name: TARGET
+              value: "World v2"
+```
+
+Now refresh the browser and you should see `Hello World v2`.
+
+## Blue-Green Deployment
+
+Deploying a new Knative service actually creates a new immutable revision and routes all traffic to it.  These immutable revisions allows us to have sophisticated deployment strategies.  The first we will examine is blue-green.
+
```
Comment (dudash) [inline workshop/content/lab1.6_operator.md:124]: Can we have something like to read more about what a blue-green deployment is, check out this link:
https://martinfowler.com/bliki/BlueGreenDeployment.html

File: workshop/content/lab1.6_operator.md [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+      percent: 100
+
+```
+
+Refresh the page and you will see the version 2 text.
+
+## Canary Deployment
+
+The immutable revisions coupled with our ability to configure traffic distribution means we can also do Canary deployment.  Let's try that now.
+
```
Comment (dudash) [inline workshop/content/lab1.6_operator.md:173]: Similar to comment above - can we have something like: If you aren't familiar with the concept of a canary deployment you can check out this link: https://martinfowler.com/bliki/CanaryRelease.html

**Review by dudash — APPROVED** [review]

**Comment by jkeam** [comment]
Related to https://github.com/RedHatGov/serverless-workshop-code/pull/10

File: workshop/content/lab1.6_operator.md (hunk 1 above)
Comment (jkeam) [reply workshop/content/lab1.6_operator.md:124]: Great idea, will add.

File: workshop/content/lab1.6_operator.md (hunk 2 above)
Comment (jkeam) [reply workshop/content/lab1.6_operator.md:173]: Great idea, will add.

File: workshop/content/lab1.6_operator.md [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+```
+curl -sS https://raw.githubusercontent.com/RedHatGov/serverless-workshop-code/deployment/hello-canary.yml | oc apply -f -
```
Comment (theckang) [inline workshop/content/lab1.6_operator.md]: minor typo: `hello-python-canary.yml`

**Review by theckang — CHANGES_REQUESTED** [review]

**Comment by theckang** [comment]
good stuff with the operations lab!  I enjoyed testing the blue/green rollouts.

File: workshop/content/lab1.6_operator.md (hunk 3 above)
Comment (jkeam) [reply workshop/content/lab1.6_operator.md]: Great catch, fixed.

File: workshop/content/lab1.6_operator.md (hunk 2 above)
Comment (jkeam) [reply workshop/content/lab1.6_operator.md:173]: Just added.

File: workshop/content/lab1.6_operator.md (hunk 1 above)
Comment (jkeam) [reply workshop/content/lab1.6_operator.md:124]: Just added.

**Review by theckang — APPROVED** [review]

**Pull request merged**